recording/lib/reporter.ex

```diff
@@ -30,11 +38,22 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.Reporter do
   end
 
   @spec add_track(pid(), Track.t(), filename(), pos_integer()) :: :ok
-  def add_track(reporter, track, filename, offset) do
-    track = Map.put(track, :offset, offset)
+  def add_track(reporter, track, filename, timestamp) do
+    track = Map.put(track, :offset, timestamp)
+
     GenServer.cast(reporter, {:add_track, track, filename})
   end
 
+  @spec start_track(pid(), Track.id(), pos_integer()) :: :ok
+  def start_track(reporter, track_id, timestamp) do
+    GenServer.cast(reporter, {:start_track, track_id, timestamp})
+  end
+
+  @spec end_track(pid, Track.id(), pos_integer()) :: :ok
+  def end_track(reporter, track_id, end_timestamp) do
+    GenServer.cast(reporter, {:end_track, track_id, end_timestamp})
+  end
+
```

Comment:
for me these names are misleading `start_track` doesn't actually start track and `end_track` doesn't end track. I would suggest something like `start_timestamp`, `track_start_timestamp`,  `end_timestamp`, or `last_timestamp`.